processing-common/src/main/kotlin/com/livefront/sealedenum/internal/common/spec/SealedEnumTypeSpec.kt

```diff
@@ -77,8 +78,9 @@ internal data class SealedEnumTypeSpec(
 
     private fun createObjectsProperty(): PropertySpec = PropertySpec.builder("values", listOfSealedClass)
         .addModifiers(KModifier.OVERRIDE)
-        .initializer(
+        .delegate(
             buildCodeBlock {
+                beginControlFlow("lazy(mode = %T.SYNCHRONIZED)", LazyThreadSafetyMode::class.asTypeName())
```

Comment:
I think we should use [`LazyThreadSafetyMode.PUBLICATION`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-lazy-thread-safety-mode/-p-u-b-l-i-c-a-t-i-o-n.html#publication) here, we don't need to protect against the list being created more than once that `SYNCHRONIZED` guarantees so we can avoid the cost of synchronizing.